src/class-parsely.php

```diff
@@ -129,6 +131,52 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	/**
+	 * Include a link to the statistics page for an article in the wp-admin Posts List.
+	 * If the post object is the "front page," this will include the main dashboard link instead.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/page_row_actions/
+	 * @see https://developer.wordpress.org/reference/hooks/post_row_actions/
+	 * @param [string[]] $actions  The list of actions to which this function will append
+	 * @param [WP_Post]  $post     The individual post object in the list
+	 * @return $actions            The filtered list of actions
+	 */
+	public function row_actions_add_parsely_link( $actions, $post ) {
+		if ( ! is_array( $actions ) ) {
+			return $actions;
+		}
+
+		if ( ! self::post_has_trackable_status( $post ) ) {
+			return $actions;
+		}
+
+		$options = $this->get_options();
+		$apikey = $options['apikey'];
+		$parsely_url = trailingslashit( 'https://dash.parsely.com/' . $apikey );
+
+		if (
+			'page' === get_option( 'show_on_front' ) &&
+			(int) get_option( 'page_on_front' ) === $post->ID
+		) {
+			// This is the "front page." Link to the dashboard.
+			$actions[ 'parsely_dash' ] = sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_url( $parsely_url ),
+				esc_html__( 'Parse.ly Dashboard', 'wp-parsely' )
+			);
```

Comment:
Why does this need to be different?

The front page still has it's own stats page on the Parse.ly dashboard i.e. `dash.parsely.com/wpvip.com/find?url=https://wpvip.com` still works.

Having a list of pages pointing to "Parse.ly Stats", and then one in the middle pointing to "Parse.ly Dashboard" seems odd. Let's keep it consistent.

Reply:
I went ahead and took out the variation.  Agree that having it consistent makes more sense after more testing.